<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="ClassifierView" name="classifierView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuTarget">
     <property name="title">
      <string>Target</string>
     </property>
     <addaction name="actionSD"/>
     <addaction name="actionSG"/>
     <addaction name="actionSD_SG"/>
    </widget>
    <addaction name="actionText_size"/>
    <addaction name="actionRects_colors"/>
    <addaction name="separator"/>
    <addaction name="menuTarget"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSplit_in_sentences"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionRun_Plotter"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionSubmenu2">
   <property name="text">
    <string>Submenu2</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSeve">
   <property name="text">
    <string>Seve</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionText_size">
   <property name="text">
    <string>Text size</string>
   </property>
  </action>
  <action name="actionRects_colors">
   <property name="text">
    <string>Rects colors</string>
   </property>
  </action>
  <action name="actionRun_Plotter">
   <property name="text">
    <string>Run Plotter</string>
   </property>
  </action>
  <action name="actionSplit_in_sentences">
   <property name="text">
    <string>Split in sentences</string>
   </property>
  </action>
  <actiongroup name="actiongroupTarget">
   <action name="actionSG">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>SG</string>
    </property>
   </action>
   <action name="actionSD">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>SD</string>
    </property>
   </action>
   <action name="actionSD_SG">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>SD/SG</string>
    </property>
   </action>
  </actiongroup>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClassifierView</class>
   <extends>QGraphicsView</extends>
   <header>classifier_view</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
